### PR TITLE
Implement method SICAD::setOglToCam

### PR DESCRIPTION
**Pull request by xEnVrE**
The class `SICAD` contains a **private** member `ogl_to_cam` deciding the relative orientation between the real camera and the OpenGL camera coordinate systems. Only one out of four constructors of `SICAD` allows deciding this parameter. 

Instead of adding the parameter to all the constructors and specifying a default, which is something we had in the past If I am not wrong, this PR adds a method `setOglToCam` for that. Although this deferred method does not assure that the user specifies his choice when construction the `SICAD` object, the user is adequately notified about the standard camera convention used in the library in the documentation of each constructor.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # 📜 SuperimposeMesh changelog
 
 ## 🔖 Version 0.11.100
-
+##### `Feature`
+ - SICAD now provide a method `setOglToCam()` that allows deciding a static transformation between the OpenGL camera and a desired reference frame.
```
Comment (claudiofantacci) [inline CHANGELOG.md]: Nit:
```
SICAD now provides a setter `setOglToCam()` for the static transformation between the OpenGL camera and the desired reference frame.
```

File: src/SuperimposeMesh/src/SICAD.cpp [hunk 2]
```diff
@@ -1288,6 +1288,15 @@ bool SICAD::setProjectionMatrix
 }
 
 
+void SICAD::setOglToCam(const std::vector<float>& ogl_to_cam)
+{
+    if (ogl_to_cam.size() != 4)
+        throw std::runtime_error("ERROR::SICAD::setOglToCam\nERROR:\n\tWrong size provided for ogl_to_cam.\n\tShould be 4, was given " + std::to_string(ogl_to_cam.size()) + ".");
```
Comment (claudiofantacci) [inline src/SuperimposeMesh/src/SICAD.cpp]: `Wrong size provided for ogl_to_cam.` -> `Wrong input size.`

**Review by claudiofantacci — APPROVED** [review]
LGTM!
Only minor details.

File: src/SuperimposeMesh/src/SICAD.cpp (hunk 2 above)
Comment (xEnVrE) [reply src/SuperimposeMesh/src/SICAD.cpp]: Done!

File: CHANGELOG.md (hunk 1 above)
Comment (xEnVrE) [reply CHANGELOG.md]: Done!

**Review by claudiofantacci — APPROVED** [review]

**Pull request merged**